.github/workflows/hostplumber-multiarch-build-and-push.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: docker
+on:
+  push:
+    branches:
+      - 'master'
+      - 'v*'
+      - 'private/**'
+    paths:
+      - hostplumber/**
+
+jobs:
+  buildx:
+    env:
+      USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
+      PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      BUILD_ARGS: ""
+      PLATFORMS: linux/amd64
+      BRANCH: ${{ github.head_ref || github.ref_name }}
+      PUSH: true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set env
+        run: 
+          echo "TAG=$(echo "${{ env.BRANCH }}" | tr -d /)" >> $GITHUB_ENV
```

Comment:
In the existing Makefile, we don't simply use the git branch as tag. We call the `get-label.sh` script [here](https://github.com/platform9/luigi/blob/master/hostplumber/Makefile#L3) from `Makefile`. 
The script uses the tag if the current git revision is tagged. Else the branch name. And then applies a suffix (either the Teamcity build ID if there is one or  the git revision SHA ID)

Reply:
What if we use ${{ github.run_number }} here instead.
Tag will be `branchnme-${{ github.run_number }}`

Reply:
Sure, the `run_number` is a good replacement for `TEAMCITY_BUILD_ID`.

Reply:
We can also modify the Makefile to use the run_number as an environment parameter and call the make target from Github action?

Reply:
Updated to call make target.